ddpui/ddpairbyte/airbytehelpers.py

```diff
@@ -717,3 +715,52 @@ def delete_source(org: Org, source_id: str):
     logger.info(f"deleted airbyte source {source_id}")
 
     return None, None
+
+def get_connection_catalog(connection_id: str):
+    """
+    Get the catalog diff of a connection.
+    """
+    res = []
+    try:
+        connection = airbyte_service.get_connection_catalog(connection_id)
+        res.append(
```

Comment:
any reason you're returning a single-element list instead of just the element?

Reply:
removed the list